app/_layouts/plugin.html

```diff
@@ -43,6 +157,53 @@ <h5>{{ nav.label }}</h5>
     <div class="page-content-container">
       <div class="page-content">
         <div class="content">
+
+          {{ page.description | markdownify }}
+
+          <hr>
+
+          <h2 id="terminology">Terminology</h2>
+          <ul>
+            <li><code>plugin</code>: a plugin executing actions inside Kong before or after a request has been proxied to the upstream API.</li>
+
+            {% if page.params.api_id %}
+              <li><code>api</code>: your upstream service placed behind Kong, for which Kong proxies requests to.</li>
+            {% endif %}
+            {% if page.params.service_id or page.params.route_id %}
+              <li><code>service</code>: The kong entity representing an external <em>upstream</em> API or microservice.</li>
+            {% endif %}
+            {% if page.params.route_id %}
+              <li><code>route</code>: The kong entity representing a way to map downstream requests to upstream services.</li>
+            {% endif %}
+            {% if page.params.consumer_id %}
+              <li><code>consumer</code>: a developer or machine using the api. When using Kong, a Consumer only communicates with Kong which proxies every call to the said, upstream api.</li>
+              <li><code>credential</code>: a unique string associated with a consumer, also referred to as an API key.</li>
+            {% endif %}
+          </ul>
+
+          <hr>
+
+          <h2 id="configuration">Configuration</h2>
+          {% if page.params.api_id %}
+            {{ plugin_config_for_api | markdownify }}
+          {% endif %}
+          {% if page.params.service_id %}
+            {{ plugin_config_for_service | markdownify }}
+          {% endif %}
+          {% if page.params.route_id %}
+            {{ plugin_config_for_route | markdownify }}
+          {% endif %}
+          {% if page.params.consumer_id %}
+            {{ plugin_config_for_consumer | markdownify }}
+          {% endif %}
+          <p>All plugins can be configured using the <code>/plugins</code> endpoint. A plugin which is not associated to any api, service, route or consumer is considered "global", and will be run on every request.</p>
```

Comment:
"and will be run on every request": I'm not a native speaker but I think it should be either "will run on" or "will be executed on"

Reply:
`run ran run` :D